packages/plugin-react/src/index.ts

```diff
@@ -70,7 +70,8 @@ export default function viteReact(opts: Options = {}): PluginOption[] {
         resolve: projectRoot
       })
       isProduction = config.isProduction
-      skipFastRefresh = isProduction || config.command === 'build'
+      skipFastRefresh =
+        skipFastRefresh || isProduction || config.command === 'build'
```

Comment:
Isn't this the same as:?

```suggestion
      skipFastRefresh ||= isProduction || config.command === 'build'
```

Reply:
Yes, although it looks like that was introduced in Node 15.  Would it be compiled down to something older versions of node would understand?  Honestly, the format I used feels a bit more clear to me, but I don't feel strongly if you want me to use the logical OR assignment.

Reply:
I think for the typescript files, yes, it will be transpiled down  
But let's @patak-js decide :slightly_smiling_face: as a third party

Reply:
+1 for `||=` here